swagger/v1/api_spec.json

```diff
@@ -516,6 +516,129 @@
         }
       }
     },
+    "/api/v1/participants/ecf": {
```

Comment:
I think we should remove `/api/v1/participants` from the Swagger docs. It's going to be _very_ confusing otherwise :D

Reply:
We could also add a note here that this is the new endpoint

Reply:
Hmmm, I could add big `DEPRECATED` letters to the old endpoint docs. 

I guess the alternative would be to say something like "also available on api/v1/participants" on the new endpoint documentation.

I'm worried that in a week a provider who hasn't upgraded yet tries to find docs for the endpoint they are using, and can't :P

Reply:
In the API reference we say we will include a DEPRECIATION header, I don't necessarily think we have to right now, just something to note

Reply:
I just added some clarification to the swagger docs - including mentioning that the old endpoints are deprecated.

@jamesrose do you reckon it makes the intention clear enough? @mattonfoot any other places where we should add deprecation notice?

Reply:
@kicferk1 IMO I'd remove the docs entirely. The endpoint will still be there and they will already know to move to the new one. Plus, it'll encourage them to move over sooner.